src/global-styles.ts

```diff
@@ -73,7 +73,7 @@ export const globalStyles = css`
 
     // Action
     --action: var(--earth400);
-    --actionBackground: var(--earth50);
+    --actionBackground: #00b6f01f;
     --onAction: var(--nova);
```

Comment:
Are we not making new variables for these, to be consistent with all other colors?

Reply:
Hmmm good point, will check with Mark!